RetailCoder.VBE/Common/ICodeModuleExtensions.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Antlr4.Runtime;
+using Rubberduck.Parsing.Symbols;
+using Rubberduck.VBEditor.SafeComWrappers.Abstract;
+
+namespace Rubberduck.Common
+{
+    public static class ICodeModuleExtensions
+    {
+        public static void ReplaceToken(this ICodeModule module, IToken token, string replacement)
+        {
+            var original = module.GetLines(token.Line, 1);
+            var result = ReplaceStringAtIndex(original, token.Text, replacement, token.Column);
+            module.ReplaceLine(token.Line, result);
+        }
+        public static void ReplaceIdentifierReferenceName(this ICodeModule module, IdentifierReference identifierReference, string replacement)
```

Comment:
Nitpick: I like to have an empty line between members of a class - that would be consistent with the rest of the code base.